src/components/Sorted.jsx

```diff
@@ -0,0 +1,50 @@
+import { useSearchParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { fetchSortOrder } from "../api";
+import ArticleCard from "./ArticleCard";
+
+export default function Sorted () {
+
+    const [searchParams, setSearchParams] = useSearchParams();
+    const [sorted, setSorted] = useState([]);
```

Comment:
Same naming problem here!

try to name your variables/state after what data it is storing